.github/workflows/tests.yml

```diff
@@ -167,6 +167,14 @@ jobs:
         brew install vale
         vale docs/
 
+    - name: Run ShellCheck
+      run: |
+        brew install shellcheck
+        shellcheck --color bin/brew
```

Comment:
This is (or at least should be) already run by `brew style`: https://github.com/Homebrew/brew/blob/081025c96a3fbd7210200f431c5110d05dd6d14c/Library/Homebrew/style.rb#L105-L115